constants.ts

```diff
@@ -6,6 +6,11 @@ export const TERRA_MAIN_NET = { URL: 'https://lcd.contco.dev', chainID: 'columbu
 export const TERRA_OBSERVER_URL = 'wss://observer.terra.dev';
 export const TERRA_SWAP_GRAPHQL_URL = TEFI_API + 'https://terraswap-graph.terra.dev/graphql';
 export const CLUB_SERVER_ROOT = 'https://tef-club-server.herokuapp.com';
+export const FCD_URL = 'https://fcd.terra.dev';
+export const LCD_URL = 'https://lcd.terra.dev';
+export const TESTNET_LCD_URL = 'https://pisco-lcd.terra.dev';
+export const DGORA_TESTNET_CONTRACT_ADDRESS = 'terra18luvq9wv88lk0h5xvx2eatnssqew2z36g7f2q0zhk4337q4g7wnqzwhe9w';
+export const MICRO = 1000000;
```

Comment:
MICRO, LCD, FCD is already added to the terra-utilities package, we can re-use them from the package.

Reply:
Done